components/aws/sagemaker/common/_utils.py

```diff
@@ -395,3 +490,30 @@ def get_best_training_job_and_hyperparameters(client, hpo_job_name):
 
 def id_generator(size=4, chars=string.ascii_uppercase + string.digits):
   return ''.join(random.choice(chars) for _ in range(size))
+
+
+def str_to_bool(s):
```

Comment:
Why not try to use built-in algo? Like `disutils` pkg and some `strtobool` methods.

Reply:
I think I wrote them for quick prototype in the past which is not good.. let's try to reuse from libraries

Reply:
Oh I actually added those in. I didn't realize there were libraries for this, but I looked up strtobool and looked for other methods to convert str to int, dicts, lists, etc. and they don't quite do what I need them to do
I could use strtobool but it would take more work in the helpers when using those values to make the request?

Reply:
OK. If not all helper can be migrate to libraries, then let's just keep it for now.